automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -116,8 +116,9 @@ object Leonardo extends RestClient with LazyLogging {
 
     def get(googleProject: GoogleProject, clusterName: ClusterName)(implicit token: AuthToken): Cluster = {
       val path = clusterPath(googleProject, clusterName)
-      logger.info(s"Get details for cluster: GET /$path")
-      handleClusterResponse(parseResponse(getRequest(url + path)))
+      val cluster = handleClusterResponse(parseResponse(getRequest(url + path)))
+      logger.info(s"GET /$path. Response: $cluster")
```

Comment:
log with details that looks like this:
```
INFO  [o.b.d.w.l.Leonardo$      :get] - GET /api/cluster/gpalloc-dev-master-2k8iu3r/automation-test-aabs4xfuz. Response: Cluster(ClusterName(automation-test-aabs4xfuz),1dde73ad-eb83-4551-a0e9-6cc701a59533,gpalloc-dev-master-2k8iu3r,ServiceAccountInfo(Some(b623pet-114763077412354570085@gpalloc-dev-master-2k8iu3r.iam.gserviceaccount.com),None),MachineConfig(Some(0),Some(n1-standard-4),Some(100),None,None,None,None),https://leonardo.dsde-dev.broadinstitute.org/notebooks/gpalloc-dev-master-2k8iu3r/automation-test-aabs4xfuz,OperationName(projects/gpalloc-dev-master-2k8iu3r/regions/us-central1/operations/11356072-944e-4261-b1e2-90df4b603ca0),Creating,None,ron.weasley@test.firecloud.org,2018-06-20T23:27:52.581Z,None,Map(notebookExtension -> gs://leonardo-swat-test-bucket-do-not-delete/, creator -> ron.weasley@test.firecloud.org, googleProject -> gpalloc-dev-master-2k8iu3r, clusterServiceAccount -> b623pet-114763077412354570085@gpalloc-dev-master-2k8iu3r.iam.gserviceaccount.com, clusterName -> automation-test-aabs4xfuz, foo -> f1aghgu),Some(GcsPath(leonardo-swat-test-bucket-do-not-delete,GcsObjectName(,1970-01-01T00:00:00Z))),None,Some(leostaging-automation-test-3a1a1d23-dd29-491d-ac80-b1e6741018c7),List())
```

instead look like this in original code:
`INFO  [o.b.d.w.leonardo.Leonardo$    :get]     - Get details for cluster: GET /api/cluster/gpalloc-dev-master-21qcecv/automation-test-a0rvv61rz`

I can revert if don't like new log

Reply:
👍